ADAL/ADAL.xcodeproj/project.pbxproj

```diff
@@ -1208,6 +1212,7 @@
 			isa = PBXFrameworksBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
+				96285D7C20D87DF8004CA4BD /* SafariServices.framework in Frameworks */,
```

Comment:
it personally feels a bit weird to include SafariServices in ADAL if system webview won't be publicly available in ADAL. Can we do some conditional compilation so that we don't force SafariServices to be included and customers won't benefit from it.